### ISSUE TITLE
Library cannot install via npm@7 together with bsb@8.3.1

### ISSUE THREAD

**Issue by wingyplus** [reasonml/reason-react#629]
Reproduce steps:

```
$ bsb -version
8.3.1
$ bsb -init react-test -theme basic
Making directory react-test
$ cd react-test/
$ ls
README.md	bsconfig.json	package.json	src/
$ npm i --save reason-react
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: react-test@0.1.0
npm ERR! Found: bs-platform@8.3.1
npm ERR! node_modules/bs-platform
npm ERR!   dev bs-platform@"^8.3.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer bs-platform@"^7.1.1" from reason-react@0.9.1
npm ERR! node_modules/reason-react
npm ERR!   reason-react@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/thanabodee/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/thanabodee/.npm/_logs/2020-10-22T05_30_39_975Z-debug.log
```

The problems cause from `reason-react` require bs-platform that lower than bs-platform in the main project. The solution might be try using `>=` in the bs-platform version.

NOTE: The current workaround is running `npm i --save --legacy-peer-deps reason-react` to install on `npm@7`.

**Comment by bbruneau** [comment]
I've been fighting with this issue, too. It would sure be nice if someone from the team were to look at this.

**Comment by davesnx** [comment]
Released a new version (0.10.0) that should fix any installation issue with latest bs-platform (v9). Let me know if you find any issues and will try to fix them. 

Thanks for reporting those and sorry for the late response/fix.
